templates/azure-pipelines.yml

```diff
@@ -92,4 +92,5 @@ if ($PLASTER_PARAM_Artifact -eq 'Yes') {
   inputs:
     codeCoverageTool: JaCoCo
     summaryFileLocation: "**/CodeCoverageResults*.xml"
-    failIfCoverageEmpty: false
\ No newline at end of file
+    pathToSources: "$(System.DefaultWorkingDirectory)\<%=$PLASTER_PARAM_ModuleName%>"
```

Comment:
@mikejwhat, have you tested this? When I tested this in my project, I got an error saying `Failed to generate Html report. Error: 1` because it couldn't find the path to the sources. It only worked for me when I changed the `\` to `/` (similar to `summaryFileLocation` property)